opm/input/eclipse/EclipseState/Grid/FieldProps.cpp

```diff
@@ -814,6 +777,52 @@ bool FieldProps::has<int>(const std::string& keyword) const {
 }
 
 
+void FieldProps::apply_multipliers()
+{
+    bool hasPorvBefore = (this->double_data.find(ParserKeywords::PORV::keywordName) == this->double_data.end());
```

Comment:
Would you mind adding a comment here for why `hasPorvBefore` is the same as `PORV` **not** being in `double_data`, please?  Also, is there a reason for not marking `hasPorvBefore` as `const`?